packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -1601,9 +1693,13 @@ describe('plugin-meetings', () => {
       });
 
       it('if newLocus replaceAt time is expired, then return false', () => {
-        sinon.stub(webex.meetings.meetingCollection, 'getActiveBreakoutLocus').returns({joinedWith: {replaces: [{
+        sinon.stub(webex.meetings.meetingCollection, 'getActiveBreakoutLocus').returns({
```

Comment:
why are we doing formatting changes? Please avoid any unnecessary formatting changes as I have a bit PR coming up for local tracks for transcoded meetings and these changes will cause a lot of conflicts

Reply:
I'll remove formatting changes.